src/include/duckdb/main/capi/header_generation/apis/v1/v1.2/v1.2.0.json

```diff
@@ -56,6 +56,10 @@
         "duckdb_param_logical_type",
         "duckdb_clear_bindings",
         "duckdb_prepared_statement_type",
+        "duckdb_nfields",
```

Comment:
These will need to be added to the unstable part of the api instead of `v1.2.0`

See `src/include/duckdb/main/capi/header_generation/apis/v1/unstable/instance_cache.json`

Reply:
Thanks! I have moved them to the unstable part.